src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -712,6 +712,9 @@ export default class ConvasController {
         if (aircraft.category === FLIGHT_CATEGORY.DEPARTURE) {
             return;
         }
```

Comment:
I'm not really following the logic, because as I understand it `aircraft.category` is in `AircraftController`. Anyway, I'll fix the newline now.